src/commands/sync.ts

```diff
@@ -124,15 +124,15 @@ export const commandSync = command({
 
     watchStats();
 
-    // Upload larger files
-    await uploadBigFiles(state, bigFiles, target);
-    // Tar small files and upload them
-    await uploadSmallFiles(state, smallFiles, target);
+    // Upload larger files and Tar small files and upload them
+    if (bigFiles.length > 0) await uploadBigFiles(state, bigFiles, target);
+    if (smallFiles.length > 0) await uploadSmallFiles(state, smallFiles, target);
 
     // Force a scan after the upload completes
     if (state.scan === false) {
```

Comment:
Is the `scan` argument still useful if we force it?

Reply:
Would skip the first upload part, I guess we could disable it completely